src/__tests__/parser.ts

```diff
@@ -1817,5 +1817,31 @@ describe('parser', () => {
         ''
       );
     });
+
+    it.only('should handle when parameters are assigned to default exports (subcomponents)', () => {
+      check(
+        'SubComponent',
+        {
+          Root: {
+            name: {
+              type: 'string',
+              defaultValue: undefined,
+              required: true,
+              description: ''
+            }
+          },
+          Sub: {
```

Comment:
Without the change below, the component names are `Root` and `default.Sub`, which is not correct.